src/mcp/tools/core/read_resources.ts

```diff
@@ -36,6 +38,7 @@ export const read_resources = tool(
 
     const out: string[] = [];
     for (const uri of uris) {
+      await trackGA4("mcp_read_resource", { resource_name: uri });
       const resolved = await resolveResource(uri, ctx);
```

Comment:
Can you move this to a `{track: true}` option on `resolveResource` that is supplied both here and on the `mcpReadResource` method? That way if MCP clients are directly reading the resource via MCP resource calls (vs. doing a tool call) we can see that too.

Reply:
Done.